python/test_MemberParser.py

```diff
@@ -87,38 +97,44 @@ def test_parse_valid(self):  # pylint: disable=too-many-statements
     self.assertTrue(parsed.is_builtin_array)
     self.assertEqual(int(parsed.array_size), 4)
     self.assertEqual(parsed.array_type, r'double')
+    self.assertEqual(parsed.julia_type, r'MVector{4, Float64}')
 
     # an array definition as terse as possible
     parsed = parser.parse(r'std::array<int,2>anArray//with a comment')
     self.assertEqual(parsed.full_type, r'std::array<int, 2>')
     self.assertEqual(parsed.name, r'anArray')
     self.assertEqual(parsed.description, r'with a comment')
+    self.assertEqual(parsed.julia_type, r'MVector{2, Int32}')
 
     parsed = parser.parse('::TopLevelNamespaceType aValidType // hopefully')
     self.assertEqual(parsed.full_type, '::TopLevelNamespaceType')
     self.assertEqual(parsed.name, r'aValidType')
     self.assertEqual(parsed.description, 'hopefully')
+    self.assertEqual(parsed.julia_type, r'TopLevelNamespaceType')
 
     parsed = parser.parse(r'std::array<::GlobalType, 1> anArray // with a top level type')
     self.assertEqual(parsed.full_type, r'std::array<::GlobalType, 1>')
     self.assertEqual(parsed.name, r'anArray')
     self.assertEqual(parsed.description, r'with a top level type')
     self.assertTrue(not parsed.is_builtin_array)
     self.assertEqual(parsed.array_type, r'::GlobalType')
+    self.assertEqual(parsed.julia_type, r'MVector{1, ::GlobalType}')
```

Comment:
Should this look like:
`self.assertEqual(parsed.julia_type, r'MVector{1, ::GlobalType}')`